llvm/lib/ProfileData/Coverage/CoverageMappingWriter.cpp

```diff
@@ -167,7 +167,18 @@ void CoverageMappingWriter::write(raw_ostream &OS) {
       return LHS.FileID < RHS.FileID;
     if (LHS.startLoc() != RHS.startLoc())
       return LHS.startLoc() < RHS.startLoc();
-    return LHS.Kind < RHS.Kind;
+
+    // Put `Decision` before `Expansion`.
+    auto getKindKey = [](CounterMappingRegion::RegionKind Kind) {
```

Comment:
Would it make sense to make this into a real function, so that its return value can be tested in mock?

Reply:
I think it'd not be required till whole reordering would come.